setup.cfg

```diff
@@ -13,13 +13,13 @@ markers =
 
 [pep8]
 max-line-length = 99
-ignore = E121,E123,E126,E133,E226,E241,E242,E704
+ignore = E121,E123,E126,E133,E226,E241,E242,E302,E704
 exclude = build/*,.tox/*,tests/*,ve/*,*/_vendor/*,auxlib/*,conda/progressbar/*,conda/compat.py
 
 
 [flake8]
 max-line-length = 99
-ignore = E121,E123,E126,E133,E226,E241,E242,E704
+ignore = E121,E123,E126,E133,E226,E241,E242,E302,E704
```

Comment:
I decided to disable E302 (expected two blank lines instead of 1) instead of going through and fixing them all. There were a _ton_ of warnings (>100), and I don't really think it helps readability.

If you'd prefer to keep it on, though, I can go back and insert all the new lines.